### PR TITLE
Add Prettier extension to remote container

### PR DESCRIPTION
**Pull request by ajyong**
`devcontainer.json` got formatted by... prettier.

### CODE REVIEW

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Cool idea @ajyong 

Can you just clarify when this will format to code? With this installed are files formatted on save, or do you need to trigger the add-on?

**Comment by ajyong** [comment]
By default it does nothing.  I think I should add some instructions in the README to:

1. Make Prettier the default formatter
1. Clarify that it formats per-file, on-save if this is enabled in VS Code

**Comment by ajyong** [comment]
It's more heavy-handed now, and I've added documentation in the README to reflect this.  By default, this PR sets the remote container to:

- Use Prettier as the **default formatter**
- Format your files **on save**
- Trim trailing whitespace from all files **except Markdown**

Give your container a rebuild and let me know if it impedes your workflow @MarkBennett, otherwise I think these "sane defaults" will increase consistency moving forward.

**Comment by MandyMeindersma** [comment]
lgtm, I will let mark decide on final approval

File: README.md [hunk 1]
```diff
@@ -20,11 +20,34 @@ This will serve the site at:
...
+Our remote container includes a few pre-installed VS Code Extensions:
+
+- [Vim](https://marketplace.visualstudio.com/items?itemName=vscodevim.vim)
+- [Ruby](https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby)
+- [VSCode Ruby](https://marketplace.visualstudio.com/items?itemName=wingrunr21.vscode-ruby)
+- [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+By default, we have set the remote container to:
+
+- Use Prettier as the **default formatter**
```
Comment (MarkBennett) [inline README.md:32]: Sounds good to me!

**Review by MarkBennett — APPROVED** [review]
These changes look like a good start with Prettier @ajyong. Let's try them out and revise if needed.

**Pull request merged**